doc/source/management/data_at_rest_encryption.rst

```diff
@@ -477,16 +461,81 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. rubric:: Redo log encryption
+
+Implemented in 8.0.16-7, the key rotation for the redo log has been redesigned to rotate the key with `SELECT rotate_system_key("percona_redo").
```

Comment:
Implemented in :rn:`8.0.16-7`

This way it will link to the release notes of the release it was implemented in.

Reply:
Added. Thanks.

Reply:
rotate_system_key is not GA yet.

Reply:
Added Experimental tag.

Reply:
Based on other discussion - removed and saved for 8.0.17